src/filters/frame_skipping_pp.py

```diff
@@ -0,0 +1,52 @@
+from src.filters.abstract_pp import AbstractPP
+from src.utils import image_utils
+from typing import List
+
+
+class frameSkippingPP(AbstractPP):
+
+    def __init__(self, threshold=0.0, compare_foreground=False, 
```

Comment:
Missing comments for what each parameter does? Names are self-explanatory but still, it is a good practice.

Reply:
Added comments for all functions